schema/codelists/documentType.csv

```diff
@@ -52,3 +52,4 @@ contract,tariffReview,Tariff review,For summarizing the arrangements for the rev
 contract,tariffIllustration,Tariff illustration,For linking to graphs and reports on the change over time in tariff prices. Use the relevant image media type when linking to PNG or JPEG or GIF graphs to allow applications to directly display this content.,,
 tender,eligibilityCriteria,Eligibility criteria,Detailed documents about the eligibility of bidders.,1.2,"This code had unclear semantics and used ambiguous terms, and is deprecated in favor of 'exclusionGrounds'."
 "tender, award, contract",contractSchedule,Contract schedules,"Any document which contains additional terms, obligations or information related to the contract, such as a schedule, appendix, annex, attachment or addendum.",1.2,Deprecated in favor of contractAnnexe.
+"tender, award",noObjectionCertificate,No objection certificate,A document issued by a relevant authority certifying that they have no objections to the tender or award decision.,,
```

Comment:
Remove the constraint on who the issuer is. Use "award" rather than "award decision".

```suggestion
"tender, award",noObjectionCertificate,No objection certificate,A document that certifies that the issuer has no objections to the tender or award.,,
```